packages/create/src/helpers.ts

```diff
@@ -383,6 +335,128 @@ async function checkPostgresDbExists(options: any, root: string): Promise<true>
     return true;
 }
 
+/**
+ * Check to see if Docker is installed and running.
+ * If not, attempt to start it.
+ * If that is not possible, return false.
+ *
+ * Refs:
+ * - https://stackoverflow.com/a/48843074/772859
+ */
+export async function isDockerAvailable(): Promise<{ result: 'not-found' | 'not-running' | 'running' }> {
+    const dockerSpinner = spinner();
+
+    function isDaemonRunning(): boolean {
+        try {
+            execFileSync('docker', ['stats', '--no-stream'], { stdio: 'ignore' });
+            return true;
+        } catch (e: any) {
+            return false;
+        }
+    }
```

Comment:
Some setups will require root privileges for the `docker stats` command so this might fail even if technically the daemon is running.